### ISSUE TITLE
Spike: investigate where to host production dApp

### ISSUE THREAD

**Issue by nfrgosselin** [gitcoinco/grants-round#153]
**timebox to half a day**

**Overview**
We need to figure out where we want to host the production version of the Round Manager dApp. It's currently on Fleek but we need to figure out if that's our long-term plan.

**Details**
- Whoever picks this up should sync with Passport to figure out how they approached their production dApp. 
- Recommendation on where to host production dApp

**Comment by thelostone-mc** [comment]
@QEDK has an alternative project to checkout for this. 
Whoever picks this up should also hit them up and evaluate that as an option

**Comment by LukeDowell** [comment]
Notes for me:
- Spheron.network

**Comment by LukeDowell** [comment]
The tl;dr of the research is that many hosts in the web3 space seem to use IPFS which will have similar pains to what we felt with using Fleek on the Passport team. Several times I'd find a host provider, click on their example app, and get a 404.

The discussion I'd like to have as a team is, "If we are trying to serve static files, why not use tried and true hosting services that can do that cheaply and effectively?"

DigitalOcean is a service I've used in my personal projects that I absolutely love. They are cheap, effective, have ready-to-go integrations with github actions and 1st-in-class documentation and tutorials. If there are marketing or perception concerns with hosting in a centralized space we should talk about that as a team.

**Comment by nfrgosselin** [comment]
Marking the spike as done — we'll reference these notes in our session next week
